doc-content/enterprise-only/springboot/bus-app_con.adoc

```diff
@@ -0,0 +1,21 @@
+[id='bus_app_{context}']
+= Red Hat Business Automation business applications
+
+Spring Framework is a Java platform that provides comprehensive infrastructure support for developing Java applications. Spring Boot is Spring's lightweight framework based on Spring Boot starters, `pom.xml` files that contain a set of dependency descriptors that you can include in your application. 
+
+Red Hat Business Automation business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. Business applications are based on Spring Boot starters. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example order management or accommodation management. 
+
+On the https://start.jbpm.org[business application],  web site you can create a Process Automation Manager, Decision Manager, or Business Optimizer business application. After you create and configure your business application, you can deploy it to an existing service or to the cloud through OpenShift. 
+
+Business applications can contain one or more of the following projects and more than one project of the same type:
+
+* Business assets (kjar): Contains business processes, rules, and forms and are easily imported into Business Central or Decision Central. COMMENT: Is this true, can they be imported in to DM.? 
```

Comment:
yes, obviously type of assets will differ but that is up to the tooling itself to either allow or not given asset type